frontend/src/pages/Onboarding.tsx

```diff
@@ -0,0 +1,172 @@
+import { useState, ChangeEvent } from "react";
+import { Button, ButtonGroup, Flex, Spacer, Input, Heading, VStack, InputGroup, Textarea, FormLabel } from '@chakra-ui/react';
+
+const Onboarding = () => {
+    let [userInfo, setUserInfo] = useState({
+        role: '',
+        bio:'',
+        expertise: '',
+        learningGoals: '',
+        experience: '',
+        jobTitle: '',
+        interests: '',
+        hourlyRate: ''
+    });
+
+    const { role } = userInfo;
+
+    const handleChange = (e: ChangeEvent<HTMLInputElement | HTMLTextAreaElement >) => {
+        setUserInfo({...userInfo, [e.target.name]: e.target.value})
+    };
```

Comment:
Excellent

put a space between the `{ ... }`

Reply:
Formatting has been changed.